certbot/plugins/standalone.py

```diff
@@ -184,7 +200,7 @@ def __init__(self, *args, **kwargs):
     def add_parser_arguments(cls, add):
         add("supported-challenges",
```

Comment:
Does this automatically get prefixed with `standalone_` somewhere, since we're calling it from the standalone plugin? Because I see that `namespace.standalone_supported_challenges` is set, instead of `supported_challenges`.